### ISSUE TITLE
Use Domains instead of IP Addresses

### ISSUE THREAD

**Issue by MessiahMetaCorp** [MessiahMetaCorp/MessiahMetaMiner#1]
trying to use domains for the explorer and web wallet instead of IP addresses and no matter how many configurations and changes in the RPC, Settings.json conf files cannot get it to match up.  

.messiahmetacorp folder is the source code
.explorer folder is where the explorer json file is settings.json

**Comment by le2Ks** [comment]
Have you  linked your domain name to the correct ip address

**Comment by MessiahMetaCorp** [comment]
yes i tried to. i wanted to use explorer.messiahmetacorp.com for the explorer and wallet.messiahmetacorp.com for the web wallet. i also changed the DNs settings to point to my VPS servers.  I have 2 nodes where i use my own VPS servers and the RPC/P2P was given by wallet builders when they created my blockchain.  Their installation guides was very basic and only included the local host IP 127.0.01 which even following their guide i was still getting errors before i wanted to change to domains and not being able to complete and get the explorer and web wallet up and running which is another issue I need to add

**Comment by le2Ks** [comment]
a NSlookup research gives negative results for both domains:
explorer.messiahmetacorp.com
wallet.messiahmetacorp.com

meaning that their are not pointing into any ip address. You must check at your domain name's provider and fix it there.
